src/amo/components/RatingsByStar/index.js

```diff
@@ -86,85 +90,104 @@ export class RatingsByStarBase extends React.Component<InternalProps> {
     );
   }
 
+  handleRouting(star: string) {
+    const { addon, location, history, clientApp, siteLang } = this.props;
+    history.push({
+      pathname: `/${siteLang}/${clientApp}/addon/${addon.slug}/reviews/`,
+      search: `?score=${star}&src=${location.query.src}`,
+    });
+  }
+
   render() {
-    const { addon, errorHandler, i18n, groupedRatings, location } = this.props;
+    const { addon, errorHandler, i18n, groupedRatings } = this.props;
     const loading = (!addon || !groupedRatings) && !errorHandler.hasError();
-
-    const linkTitles = {
-      /* eslint-disable quote-props */
-      '5': i18n.gettext('Read all five-star reviews'),
-      '4': i18n.gettext('Read all four-star reviews'),
-      '3': i18n.gettext('Read all three-star reviews'),
-      '2': i18n.gettext('Read all two-star reviews'),
-      '1': i18n.gettext('Read all one-star reviews'),
-      /* eslint-enable quote-props */
+    const self = this;
```

Comment:
Why is this needed?

Reply:
Beacuse I have to use `this` in `createLink` function. As, `this` will be undefined in the scope of that function.